src/algebra/big_operators/basic.lean

```diff
@@ -336,6 +336,11 @@ lemma prod_hom [comm_monoid γ] (s : finset α) {f : α → β} (g : β → γ)
   (∏ x in s, g (f x)) = g (∏ x in s, f x) :=
 ((monoid_hom.of g).map_prod f s).symm
 
+@[to_additive]
+lemma prod_hom' [comm_monoid γ] (s : finset α) {f : α → β} (g : β →* γ) :
+  (∏ x in s, g (f x)) = g (∏ x in s, f x) :=
+((monoid_hom.of g).map_prod f s).symm
```

Comment:
```suggestion
(g.map_prod f s).symm
```

Which raises the question "do we really need this lemma", as its just the symmetric version of one that exists already

Reply:
Thanks. I've removed it.